llvm/lib/Target/RISCV/RISCVProfiles.td

```diff
@@ -0,0 +1,204 @@
+//===------ RISCVProfiles.td - RISC-V Profiles -------------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+class RISCVProfile<string name, list<SubtargetFeature> features>
+    : SubtargetFeature<name, "RISCVProfile", NAME,
```

Comment:
Is the third argument used anywhere? If not, can we set it to "true" to make it consistent to most of other features?

Reply:
Otherwise we have to define a member variable in `RISCVSubtarget`, even though, the subtarget emitter can't recognize the variable called for example: RVI20U32.

Reply:
And we can remove `RISCVProfileEnum` and `RISCVProfile`